### ISSUE TITLE
Web panel high connection requests per user?

### ISSUE THREAD

**Issue by dushLK** [UnamSanctam/UnamWebPanel#300]
Hi, Is there a way to limit the requests sent to panel from miners? I see thousands of requests per day/each user. I mean limiting the update check once a hour or once a day per miner?

Thanks

**Comment by UnamSanctam** [comment]
Not right now since that wouldn't be very helpful for a web panel (since the data would barely update), if you just want to change the configuration then you can use the "Remote Configuration" feature.

**Comment by dushLK** [comment]
Thanks for the reply. An option to set API CONNECTION FREQUENCY would be nice in the future builds.

**Comment by UnamSanctam** [comment]
Yes, the problem is that the web panel calculates the "Offline" status as not having connected for the last 3 minutes (so that would have to be configurable I guess), and all the other data will also not be very usable (due to the infrequent updates) so at that point it might not be much point in using the web panel.
